### PR TITLE
Add David J Harding to contributors

### PR DESCRIPTION
**Pull request by dharding**

### CODE REVIEW

**Review by jasnell — CHANGES_REQUESTED** [review]
The contributors lists the members of the @covidgreen/members team (folks with commit bits that aren't in the leads or core team). That's done as part of the onboarding process once you've been nominated and approved as a contributor. You can self nominate yourself as a contributor by opening an issue that includes a bit of background of yourself and the contributions you've made or plan to make. Generally, however, we need to see some activity before accepting self-nominations

**Comment by jasnell** [comment]
(btw, the "Request Changes" is temporary here... will keep this open until we can go through a contribution nomination... and sorry for the formality, just trying to bootstrap the governance processes ;-) ..)

**Comment by jasnell** [comment]
Nomination thread here: https://github.com/covidgreen/covid-green-app/issues/19

**Review by jasnell — APPROVED** [review]

**Review by floridemai — APPROVED** [review]

**Pull request merged**
